sql/spark-sql-example.ipynb

```diff
@@ -0,0 +1,216 @@
+      " |-- maritalStatus: struct (nullable = true)\n",
+      " |    |-- coding: array (nullable = true)\n",
+      " |    |    |-- element: struct (containsNull = true)\n",
+      " |    |    |    |-- code: string (nullable = true)\n",
+      " |    |    |    |-- display: string (nullable = true)\n",
+      " |    |    |    |-- display_aa: string (nullable = true)\n",
+      " |    |    |    |-- system: string (nullable = true)\n",
+      " |    |-- text: string (nullable = true)\n",
+      " |    |-- text_aa: string (nullable = true)\n",
+      " |-- multipleBirthBoolean: boolean (nullable = true)\n",
+      " |-- multipleBirthInteger: long (nullable = true)\n",
+      " |-- name: array (nullable = true)\n",
+      " |    |-- element: struct (containsNull = true)\n",
+      " |    |    |-- family: string (nullable = true)\n",
+      " |    |    |-- family_aa: string (nullable = true)\n",
+      " |    |    |-- given: array (nullable = true)\n",
+      " |    |    |    |-- element: string (containsNull = true)\n",
+      " |    |    |-- given_aa: array (nullable = true)\n",
+      " |    |    |    |-- element: string (containsNull = true)\n",
+      " |    |    |-- prefix: array (nullable = true)\n",
+      " |    |    |    |-- element: string (containsNull = true)\n",
+      " |    |    |-- prefix_aa: array (nullable = true)\n",
+      " |    |    |    |-- element: string (containsNull = true)\n",
+      " |    |    |-- suffix: array (nullable = true)\n",
+      " |    |    |    |-- element: string (containsNull = true)\n",
+      " |    |    |-- suffix_aa: array (nullable = true)\n",
+      " |    |    |    |-- element: string (containsNull = true)\n",
+      " |    |    |-- use: string (nullable = true)\n",
+      " |-- resourceType: string (nullable = true)\n",
+      " |-- telecom: array (nullable = true)\n",
+      " |    |-- element: struct (containsNull = true)\n",
+      " |    |    |-- system: string (nullable = true)\n",
+      " |    |    |-- use: string (nullable = true)\n",
+      " |    |    |-- value: string (nullable = true)\n",
+      " |    |    |-- value_aa: string (nullable = true)\n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "import spark.implicits._\n",
+       "path: String = /home/jovyan/work/etl/output/annotated/Patient.ndjson\n",
+       "Patient: org.apache.spark.sql.DataFrame = [address: array<struct<city:string,city_aa:string,country:string,country_aa:string,extension:array<struct<parent:string,url:string,valueDecimal:double>>,line:array<string>,line_aa:array<string>,postalCode:string,postalCode_aa:string,state:string,state_aa:string>>, birthDate: string ... 15 more fields]\n"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "// git clone https://github.com/sync-for-science/a3 \n",
+    "// docker run --rm -p 8888:8888 -v $(pwd):/home/jovyan/work jupyter/all-spark-notebook\n",
```

Comment:
This bit is the actual "how do I run it" instructions. Maybe worth pulling out into a README ;-)